Dockerfile-kubebuilder

```diff
@@ -1,10 +1,8 @@
-FROM golang:1.16 as builder
+FROM golang:1.20 as builder
 WORKDIR /go/src/app
-RUN os=$(go env GOOS) &&\
-    arch=$(go env GOARCH) &&\
-    curl -sL https://github.com/kubernetes-sigs/kubebuilder/releases/download/v2.3.2/kubebuilder_2.3.2_${os}_${arch}.tar.gz | tar -xz -C /tmp/ &&\
-    mv /tmp/kubebuilder_2.3.2_${os}_${arch} /usr/local/kubebuilder &&\
-    export PATH=$PATH:/usr/local/kubebuilder/bin
+
+ENV PATH=$PATH:/go/src/app/.bin
+
 COPY . .
 RUN make test &&\
     make manager
```

Comment:
possible improvement: in the runner image change the base image to `nonroot` and remove the `USER` clause to use the standard user instead :)